settings.sample.py

```diff
@@ -73,3 +73,9 @@
 # LINTRC_DEFAULTS = './lintrc_defaults.ini'
 
 OK_COMMENT = ':+1: No lint errors found.'
+
+# Set to True to use a label instead of a comment for OK status.
```

Comment:
Ideally lint-review could create the label if its missing. That would cut down on the work to integrate lint-review with a project.